Rock.Rest/Controllers/FinancialTransactionsController.Partial.cs

```diff
@@ -317,7 +317,33 @@ public DataSet GetContributionTransactions( int groupId, int? personId, [FromBod
         }
 
         /// <summary>
-        /// 
+        /// Gets the contribution transactions.
+        /// </summary>
+        /// <param name="groupId">The group unique identifier.</param>
+        /// <returns></returns>
+        /// <exception cref="System.Web.Http.HttpResponseException"></exception>
+        [Authenticate, Secured]
+        [HttpGet]
+        [EnableQuery]
+        [System.Web.Http.Route( "api/FinancialTransactions/GetTransactionsForGivingGroup/{personId}" )]
+        public IQueryable<FinancialTransaction> GetTransactionsForGivingGroup( int personId )
+        {
```

Comment:
@benwiley, Does your app already depend on this method/parameter? A better name for the method might be `GetByPersonGivingId` since that is what is being used to query the transactions (there is also a GivingGroupId on person which is why it's a bit confusing).  

Also, if the calling app already has a person object (with a GivingId value already), an even better idea would be to just pass the GivingId instead of PersonId and call the method `GetByGivingId( int givingId )`